app/_how-tos/create-a-super-admin.md

```diff
@@ -1,22 +1,109 @@
 ---
-title: Create a super-admin 
+title: Create a Super Admin with the Admin API
 content_type: how_to
 
 products:
     - gateway
-
+tier: enterprise
+rbac: true
 works_on:
     - on-prem
 
-
 tldr: 
-  q: How do I apply multiple rate limits or window sizes with one plugin instance?
+  q: How do I create a {{site.base_gateway}} Super-Admin using the Admin API
   a: |
-    You can use the Rate Limiting Advanced plugin to apply any number of rate limits and window sizes per plugin instance. 
-    This lets you create multiple rate limiting windows, for example, rate limit per minute and per hour, and per any arbitrary window size.
+    After enabling [RBAC](/gateway/entities/rbac/#enable-rbac), you can create a Super-Admin user by issuing a `POST` request to the [`/rbac/users/`](/api/gateway/admin-ee/#/operations/post-rbac-users) endpoint. Then associate the user to the `super-admin` role.
+
+prereqs:
+    inline:
+    #   - title: Kong Gateway running with RBAC enabled
+    #     include_content: prereqs/enable-rbac
+    #     icon_url: /assets/icons/file.svg
+      - title: Configure environment variables
+        content: |
+            Set the `kong-admin-token`, `admin_name`, and `admin-token`, for example: 
+            ```sh
+            export KONG_ADMIN_TOKEN=kong
+            export ADMIN_NAME=tim
+            export USER_TOKEN=my-admin-token
```

Comment:
maybe explain what these are for? especially the admin token, it might not be obvious that it's the password we defined in the previous prereq